### PR TITLE
Update crictl.yaml runtime-endpoint config to resolve warning

### PR DESCRIPTION
**Pull request by jcpowermac**
Original PR: https://github.com/openshift/openshift-ansible/pull/11455

Bug 1697679 - https://bugzilla.redhat.com/show_bug.cgi?id=1697679

### CODE REVIEW

File: roles/container_runtime/tasks/package_crio.yml [hunk 1]
```diff
@@ -57,6 +57,12 @@
     src: crio.conf.j2
     backup: yes
 
+- name: Update crictl.yaml runtime-endpoint
+  yedit:
+    src: /etc/crictl.yaml
+    key: runtime-endpoint
+    value: "unix://{{ openshift_crio_var_sock }}"
```
Comment (vrutkovs) [inline roles/container_runtime/tasks/package_crio.yml]:
```suggestion
    value: "unix://{{ openshift_crio_var_sock }}"
  when: not openshift_crio_var_sock.startswith('unix://')
```

File: roles/openshift_node/tasks/upgrade.yml [hunk 2]
```diff
@@ -81,6 +81,14 @@
   - "/etc/cni/net.d/100-crio-bridge.conf"
   when: crio_update is changed
 
+- name: Ensure crictl.yaml runtime-endpoint is updated
+  yedit:
+    src: /etc/crictl.yaml
+    key: runtime-endpoint
+    value: "unix://{{ openshift_crio_var_sock }}"
+  when:
+  - openshift_use_crio | default(False) | bool
```
Comment (vrutkovs) [inline roles/openshift_node/tasks/upgrade.yml]:
```suggestion
  - openshift_use_crio | default(False) | bool
  - not openshift_crio_var_sock.startswith('unix://')
```

File: roles/container_runtime/tasks/package_crio.yml (hunk 1 above)
Comment (jcpowermac) [reply roles/container_runtime/tasks/package_crio.yml]: @vrutkovs could we do something like this instead?

```yaml
set_fact:
  crio_var_sock: "{% if not openshift_crio_var_sock.startswith('unix://') %}unix://{{ openshift_crio_var_sock}}{% else %}{{ openshift_crio_var_sock}}{% endif %}"
```
That way if `openshift_crio_var_sock` was set and started with `unix://` the task wouldn't be skipped.

Comment (vrutkovs) [reply roles/container_runtime/tasks/package_crio.yml]: That looks a bit too complex, but might work, yes

**Pull request merged**